docs/refs.bib

```diff
@@ -56,7 +157,6 @@ @inproceedings{selbst2019fairness
 publisher = {Association for Computing Machinery},
 address = {New York, NY, USA},
 url = {https://doi.org/10.1145/3287560.3287598},
-doi = {10.1145/3287560.3287598},
```

Comment:
Is the DOI wrong?

Reply:
DOI is correct, but I changed it since the DOI caused a repetition of links, like:
<img width="679" alt="Screenshot 2023-01-26 at 15 42 30" src="https://user-images.githubusercontent.com/35764158/214865066-10763221-7b6b-4c20-9b40-05279814925b.png">

Should we keep it?

Reply:
I'm in favor of dropping DOI (here and elsewhere...), for exactly this reason.

Reply:
Awesome, resolving this comment then.